src/media/icon.test.jsx

```diff
@@ -68,7 +68,7 @@ describe('Icon', () => {
 		});
 
 		it('renders small shape variant for "xs" icons', () => {
-			const xsIconShape = 'test';
+			const xsIconShape = 'location';
```

Comment:
This icon has changed in the latest swarm icons build it's now `location-pin` i believe